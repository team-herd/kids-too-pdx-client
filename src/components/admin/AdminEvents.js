import React from 'react';
import PropTypes from 'prop-types';

function EventsList({ events }) {
  const listOfEvents = events.map(event => {
    return <li key={event._id}></li>;
  });

  return (
    <>
      <ul>{listOfEvents}</ul>
    </>
  );
}
EventsList.propTypes = {
  events: PropTypes.array.isRequired
};

export default EventsList;
