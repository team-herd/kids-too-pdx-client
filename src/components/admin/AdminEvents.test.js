import React from 'react';
import renderer from 'react-test-renderer';
import AdminEvents from './AdminEvents';

describe('AdminEvents', () => {
  it('matches a snapshot for admin events', () => {
    const events = [
      {
        _id: 1,
        venue: 'My House',
        name: 'House Event'
      },
      {
        _id: 2,
        venue: 'My Other House',
        name: 'Other House Event'
      },
      {
        _id: 3,
        venue: 'My Another Other House',
        name: 'Another Other House Event'
      }
    ];

    const tree = renderer.create(
      <AdminEvents events={events} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
