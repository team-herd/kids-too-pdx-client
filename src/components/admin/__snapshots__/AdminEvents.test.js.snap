// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminEvents matches a snapshot for admin events 1`] = `
<ul>
  <li />
  <li />
  <li />
</ul>
`;
